Mathlib/Analysis/SpecialFunctions/Elliptic/Weierstrass.lean

```diff
@@ -580,4 +586,299 @@ lemma derivWeierstrassP_sub_coe (z : ℂ) (l : L.lattice) :
 
 end derivWeierstrassP
 
+section AnalyticweierstrassPExcept
```

Comment:
Capitalization is inconsistent here (no big deal of course)

Reply:
It's a big deal to me so thanks for catching this :)